<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OutliNE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #282828;
	color: white;
}

QMenuBar::item::selected {
	background-color: #323232;
}

QMenuBar::item::pressed {
	background-color: #3c3c3c;
}

QMenu::item::selected {
	background: #323232;
}

QMainWindow::separator {
	background: gray;
	width: 1px;
	height: 1px;
}

QPushButton {
	background-color: #323232;
	border: 1px solid gray;
	padding: 4px;
	border-radius: 4px;
}

QPushButton:hover {
	background-color: #3c3c3c;
}

QPushButton:pressed {
	background-color: #323232;
}

QTabWidget::pane {
	border: 0px;
}

QTabBar::tab {
	background: #282828;
	border: 1px solid gray;
}

QTabBar::tab:top {
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	padding: 2px 4px;
}
QTabBar::tab:bottom {
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
	padding: 2px 4px;
}
QTabBar::tab:left {
	border-top-right-radius: 4px;
	border-bottom-right-radius: 4px;
	padding: 4px 2px;
}
QTabBar::tab:right {
	border-top-left-radius: 4px;
	border-bottom-left-radius: 4px;
	padding: 4px 2px;
}

QTabBar::tab:hover {
	background: #323232;
}

QTabBar::tab:selected {
	background: #282828;
}

QTabBar::tab:selected:top {
	border-bottom-color: #282828;
	/*margin-left: -1px;
	margin-right: -1px;*/
}
QTabBar::tab:selected:bottom {
	border-top-color: #282828;
	/*margin-left: -1px;
	margin-right: -1px;*/
}
QTabBar::tab:selected:left {
	border-left-color: #282828;
	/*margin-top: -1px;
	margin-bottom: -1px;*/
}
QTabBar::tab:selected:right {
	border-right-color: #282828;
	/*margin-top: -1px;
	margin-bottom: -1px;*/
}

QTabBar::tab:!selected:top {
	margin-top: 2px;
}
QTabBar::tab:!selected:bottom {
	margin-bottom: 2px;
}
QTabBar::tab:!selected:left {
	margin-right: 2px;
}
QTabBar::tab:!selected:right {
	margin-left: 2px;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="mainView">
      <property name="styleSheet">
       <string notr="true">QGraphicsView {
	background-color: black;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QDockWidget" name="settings">
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>95</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>252</width>
          <height>310</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Edit</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="text">
               <string>Text!</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="control">
           <property name="title">
            <string>Control</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Print</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="net">
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>95</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Net</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QGraphicsView" name="graphicsView">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="performance">
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>38</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Performance</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="instances">
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>95</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Instances</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTreeView" name="treeView">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
